results/google__text-embedding-005/external/model_meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "google/text-embedding-005",
+    "revision": "external",
+    "release_date": "2024-11-18",
+    "languages": [
+        "eng-Latn"
+    ],
+    "n_parameters": null,
+    "memory_usage": null,
+    "max_tokens": 2048.0,
+    "embed_dim": 768,
+    "license": null,
+    "open_weights": false,
+    "public_training_data": null,
+    "public_training_code": null,
+    "framework": [
+        "API"
+    ],
+    "reference": null,
```

Comment:
No release reference?

Reply:
Thanks for the catch! Added a reference to the text embedding documentation.